ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -122,6 +123,16 @@ export async function submitSelectedMhr (
     }
   }
 
+  if (shouldCallback || selected.length >= 75) {
```

Comment:
If we are putting the logic here to enforce the callback based on arr length, do we need the argument to do it manually?

Reply:
I leave it there because it is optional for sizes that are less than 75?